scripts/generateESLintTest.js

```diff
@@ -0,0 +1,44 @@
+/**
```

Comment:
One step means less developer friction. Also, `eslint .` doesn't add the output to the browsers test output. EmberJS has this as a built-in feature, and it was one of the most convenient things working with their testing infrastructure. The main thing for me is why add an additional step, and an additional place to check output, when we can do it all in one place. For me, it's a nice to have, and makes it easier to work on this project. It reduces developer friction overall, weird or not.